network-api/networkapi/wagtailpages/templatetags/localization.py

```diff
@@ -0,0 +1,22 @@
+from django import template
+
+register = template.Library()
+
+DEFAULT_LOCALE = 'en_US'
+
+mappings = {
```

Comment:
@TheoChevalier While we don't have the ability to do specific locales right now, is this set what we generally translate to by default for foundation.mozilla.org right now?

Reply:
We support the following locales `fr`, `es`, `de`, `pl`, `pt-BR`.
For French and Spanish in particular, it’s important not to specify a country code, we’re serving international (neutral) versions of the languages. If that’s not possible to use that (I think you’re saying that’s the case?), then yes, `fr-FR` and `es-ES` are the biggest markets we address.

However, I would change `pt-PT` to `pt-BR`

Reply:
Does it matter if we use underscore or dash to connect language code and region code?

Reply:
What’s the purpose of this list, is it only used to populate the og tags? Or do we have plans to use it for other things? If the former, then I assume Facebook dictates what format should be used.
However, for Mozilla purposes, I’d recommend to use a dash

Reply:
Yea it seems like using underscore for `og:locale` is the standard.

Reply:
this is purely for opengraph's `og:locale` metadata.